doc/source/examples/cfd-dem/square-fluidized-bed/square-fluidized-bed.rst

```diff
@@ -0,0 +1,392 @@
+~~~~~~~~~~~~~~~~~~~
+
+We need to pack the particles in the middle of the cylinder. Therefore, we create a stopper (floating wall) somewhere below the center of the bed. We chose the point with a y-coordinate of -0.06 to create the wall. We then define a normal to the wall at this point. Make sure that the end time of the floating wall is bigger than the simulation time to ensure that the particles remain suspended. This is shown in:
+
+.. code-block:: text
+
+    subsection floating walls
+      set number of floating walls	                = 1
+    	subsection wall 0
+		subsection point on wall
+			set x				= 0
+			set y				= -0.06
+			set z				= 0
+		end
+		subsection normal vector
+			set nx				= 0	
+			set ny				= 1
+ 			set nz 				= 0
+		end
+		set start time				= 0
+		set end time				= 5
+   	end
+
+    end
+    
+Running the simulation
+----------------------
+Launching the simulation is as simple as specifying the executable name and the parameter file. Assuming that the ``dem_3d`` executable is within your path, the simulation can be launched on a single processor by typing:
+
+.. code-block:: text
+
+  dem_3d packing_in_circle.prm
+
+or in parallel (where 8 represents the number of processors)
+
+.. code-block:: text
+
+  mpirun -np 8 dem_3d packing_in_circle.prm
+
+Lethe will generate a number of files. The most important one bears the extension ``.pvd``. It can be read by popular visualization programs such as `Paraview <https://www.paraview.org/>`_. 
+
+
+.. note:: 
+    The vtu files generated by Lethe are compressed archives. Consequently, they cannot be postprocessed directly. Although they can be easily post-processed using Paraview, it is sometimes necessary to be able to work with the raw data. The python library `PyVista <https://www.pyvista.org/>`_  allows us to do this. 
+
+After the particles have been packed inside the square bed, it is now possible to simulate the fluidization of particles.
+
+The CFD simulation is to be carried out using the packed bed simulated in the previous step. We will discuss the different parameter file sections. The mesh section is identical to that of the DEM so it will not be shown here.
+
+Simulation control
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+The simulation was run for 1 s with a time step of 0.002 s. The time scheme chosen for the simulation is first order backward difference method (BDF1). The simulation control section is shown:
+
+.. code-block:: text
+
+    subsection simulation control
+        set method                          = bdf1
+        set number mesh adapt               = 0
+        set output name                     = result_
+        set output frequency	  	    = 10
+        set startup time scaling            = 0.6
+        set time end                        = 1
+        set time step                       = 0.002
+        set subdivision                     = 1 
+        set log precision                   = 10 
+        set output path                     = ./output/
```

Comment:
Can you remove the parameters which you don't need?
If you use the output path, add a warning to encourage people to create the folder.

Reply:
Additionnaly, I would suggest pre-creating the folder with a dummy file into them so that they will already be ready to run.